app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -105,6 +140,58 @@ private void setUpToolbar() {
         }
     }
 
+    @Subscribe
+    public void tracksDownloadDone(TracksDownloadEvent event) {
+        FragmentManager fragmentManager = getSupportFragmentManager();
```

Comment:
What is this? You can't do this. You cannot change the fragment if the download succeeded.